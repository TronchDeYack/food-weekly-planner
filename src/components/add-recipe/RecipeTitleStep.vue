<template>
  <div>
    <StepTitle :step="step" :title="title" />
    <StepContent :step="step">
      <v-text-field :label="$t('addRecipe.steps.titleLabel')" />
    </StepContent>
  </div>
</template>

<script>
import StepTitle from '@/components/add-recipe/StepTitle.vue';
import StepContent from '@/components/add-recipe/StepContent.vue';

export default {
  name: 'add-recipe-step-recipe-title',
  components: {
    StepContent,
    StepTitle,
  },
  props: {
    title: {
      type: String,
    },
    step: {
      type: Number,
    },
  },
};
</script>
