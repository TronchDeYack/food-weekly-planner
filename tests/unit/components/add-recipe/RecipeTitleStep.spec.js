import RecipeTitleStep from '@/components/add-recipe/RecipeTitleStep.vue';

import { createComponent } from '../../../utils';

describe('RecipeTitleStep', () => {
  const propsData = {
    title: 'EAT ME',
    step: 2,
    complete: false,
  };

  describe('render', () => {
    it('Should render.', () => {
      const wrapper = createComponent(RecipeTitleStep, propsData);
      expect(wrapper).toMatchSnapshot();
    });
  });
});
