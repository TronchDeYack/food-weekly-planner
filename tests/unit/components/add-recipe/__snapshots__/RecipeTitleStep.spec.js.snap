// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecipeTitleStep render Should render. 1`] = `
<div>
  <steptitle-stub title="EAT ME" step="2"></steptitle-stub>
  <stepcontent-stub step="2">
    <v-text-field-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" backgroundcolor="" label="addRecipe.titleLabel" loaderheight="2" clearicon="$clear" type="text"></v-text-field-stub>
  </stepcontent-stub>
</div>
`;
